### PR TITLE
Adding i18n system to ikegami and chinese translation

### PR DESCRIPTION
**Pull request by oott123**
Hi hibiyasleep!

Sorry for this huge PR.

In this PR I have:

* Adding `yarn.lock` so that others can install the correct webpack version,
* Replaced almost all of strings that should be translated into i18n-able messages,
* Providing a self translated Chinese version of messages, and
* Adding an option for users to select between languages.

There are also some small fixes that I may forgot to mention about, sorry for that again.

This PR is ready for review. If you can review and merge my PR, I'll very happy to answer any questions.

Thank you!

### CODE REVIEW

**Comment by hibiyasleep** [comment]
in first of all, sorry, lack of i18n was intentional, since I had suffered on kagerou, as I can't write strings in all possible languages, this caused imperfection (or glitchy) on the user interface. I speaks `ko_KR`, `en_US`, `ja_JP`, and very little of `zh_TW` (traditional) as I can write little 漢字, but there were `zh_CN`, `fr_FR`, `de_DE` remained. they were almost not maintained for long time still.

tldr: code has no problem, i18n is hard, since I can't speak all of them. I saw this PR just now (very sorry!!), so I need a time to think it'll be okay or not.

**Comment by oott123** [comment]
IMO it's okay to left untranslated strings in English, and left them to be translated by others, that's how community works!
